requirements.txt

```diff
@@ -1,6 +1,6 @@
-ansible==2.2.2.0
```

Comment:
As noted in the PR description, unpinning is just to see green tests with latest Ansible (2.3), to demonstrate that the original issue is being addressed.

I'm not opposed to leaving versions pinned. One observation, though, is that it would be reasonable to verify that the versions we pin are available on the systems we support (RHEL, CentOS, ...), because even though we install these using `pip` in tests, we, in many contexts, expect users to install the dependencies via `rpm`.